eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -213,50 +213,60 @@ public async Task<int> RunTestsAsync()
             try
             {
                 // Timeout test run 5 minutes before the Helix job would timeout
-                var cts = new CancellationTokenSource(Options.Timeout.Subtract(TimeSpan.FromMinutes(5)));
+                var testProcessTimeout = Options.Timeout.Subtract(TimeSpan.FromMinutes(5));
+                var cts = new CancellationTokenSource(testProcessTimeout);
                 var diagLog = Path.Combine(Environment.GetEnvironmentVariable("HELIX_WORKITEM_UPLOAD_ROOT"), "vstest.log");
-                var commonTestArgs = $"test {Options.Target} --diag:{diagLog} --logger:xunit --logger:\"console;verbosity=normal\" --blame \"CollectHangDump;TestTimeout=15m\"";
+                var commonTestArgs = $"test {Options.Target} --diag:{diagLog} --logger xunit --logger \"console;verbosity=normal\" " +
+                                    "--blame-crash --blame-hang-timeout 15m";
```

Comment:
Does `--blame-hang-timeout` imply `--blame-hang` @nohwnd  @MarcoRossignoli❔

Reply:
Going through all my missed notifications. Yes it does. It enables --blame (creating the sequence xml file) and hang dumps with the default dump type (full), and non-default timeout (what you define for the --blame-hang-timeout). Same goes for ---blame-hang-dump-type, or --blame-crash-dump-type implying --blame-crash.